src/store/getters.ts

```diff
@@ -496,7 +497,13 @@ export const getEntityTypesBC = (state: StateIF): EntityI[] => {
     const generateEntities = (entities) => {
       const output = []
       for (const entity of entities) {
-        const obj = EntityTypesBcData.find(ent => ent.value === entity)
+        let obj = null as EntityI
+        // continuation in uses a different list than the others
+        if (isContinuationIn(state) && isLocationBC(state)) {
+          obj = EntityTypesContInData.find(ent => ent.value === entity)
+        } else {
+          obj = EntityTypesBcData.find(ent => ent.value === entity)
+        }
```

Comment:
@JazzarKarim , @jamespaologarcia :

I don't love this implementation. An alternative is to keep using the EntityTypesBcData list, and the "non-cont in" entity types (eg, ULC instead of CUL), but then swap the entity types later in the processing. What do you think? Which makes the most sense to you?

Without seriously refactoring a lot of stuff, we're kind of stuck with needing to toggle between "BC" and "cont in" types _somewhere_.

Reply:
I'm OK with this as is. I'm thinking that since this is a very special case which I don't think will re-occur, we can keep as is here. 

Now, if we stumble upon something like this in the future, we might want to reconsider but we don't want to have lots of conditions here. Thoughts? I'm OK with the alternative as well.

Reply:
I think the readability makes up for the potential code duplication because it makes it clear right away why you're changing entity types.

Reply:
Great point, James. Thanks.

Remember this when you following this architecture for your next project 😉